src/utils/getTypeString.ts

```diff
@@ -134,3 +154,66 @@ export function generateLiteralTypeString(
     }
   }
 }
+
+function instanceOfNonRecursivePP(type: TypeNode): boolean {
+  if (
+    type instanceof AddressType ||
+    type instanceof BoolType ||
+    type instanceof BuiltinStructType ||
+    type instanceof BuiltinType ||
+    type instanceof BytesType ||
+    type instanceof FixedBytesType ||
+    type instanceof ImportRefType ||
+    type instanceof IntLiteralType ||
+    type instanceof IntType ||
+    type instanceof ModuleType ||
+    type instanceof RationalLiteral ||
+    type instanceof StringLiteralType ||
+    type instanceof StringType
+  )
+    return true;
+  return false;
+}
```

Comment:
I think that code readability would improve if instead of:
```
if (condition)
    return true;
return false;
```
change it to:
```
return condition;
```